contrib/for-tests/volumes-tester/gluster/Dockerfile

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:14.04
+FROM centos
 MAINTAINER Jan Safranek, jsafrane@redhat.com
-ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update -qq && apt-get install -y glusterfs-server -qq
+ADD gluster.repo /etc/yum.repos.d/
```

Comment:
gluster stuff is unrelated to this PR description - mistake or insufficient description?